docs/reference/cluster/stats.asciidoc

```diff
@@ -38,6 +38,278 @@ include::{docdir}/rest-api/common-parms.asciidoc[tag=flat-settings]
 
 include::{docdir}/rest-api/common-parms.asciidoc[tag=timeoutparms]
 
+[[cluster-stats-api-response-body]]
+==== {api-response-body-title}
+
+`_nodes.total`::
+(integer)
+Total number of nodes in the cluster.
+
+`_nodes.successful`::
+(integer)
+Number of nodes that started succesfully.
+
+`_nodes.failed`::
+(integer)
+Number of nodes that failed to start.
+
+`cluster_name`::
+(string)
+Name of the cluster.
+
+`cluster_uuid`::
+(string)
+Unique identifier for the cluster.
+
+`timestamp`::
+(https://en.wikipedia.org/wiki/Unix_time[Unix timestamp])
+Last time the cluster statistics were refreshed.
+
+`status`::
+(string)
+Health status of the cluster based on the state of its primary and replica
+shards. Returned values include:
+
+  `green`:::
+  All primary shard and replica shards are assigned. There are at least two
+  copies of all shards.
```

Comment:
"at least two copies of all shards" isn't true - if an index has no replicas then there is only at most one copy of each shard but this index will report `green` health as long as all the primaries are assigned.

Reply:
Fixed with d0fb40f4f5b634a792dc3029ee258043797fe402.